code-challenges.test.js

```diff
@@ -14,56 +14,202 @@
+      
+    })
+  })
+//  //  ReferenceError: capitalized is not defined
+
+//  // b) Create the function that makes the test pass.
+
+  // Pseudo----->
+// create a function named capitalized
+// pass in an array paramater to the function
+// iterate over the array passed in using the .map() method
+// return the first index of the array setting it toUpperCase()
+// add the capitalized index back to the rest of the array 
+// access the rest of the string using the substring(1) method taking the 1 index  to the last index to be added to index 0
+// set the rest of the string to lowercase to ensure there are no uppercase letters using toUpperCase()
+// if outside of jest testing log the function with the desired array to be passed in
+
+  const capitalized = (array) => array.map(letter =>{
+    return letter[0].toUpperCase() + letter.substring(1).toLowerCase()
+  })
+  
+
+// capitalized
+// ✓ passes an array as a paramater and retruns each word capitalized in the array (1 ms)
 
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const vowelTester1 = "learn"
-// Expected output: 1
-const vowelTester2 = "academy"
-// Expected output: 0
-const vowelTester3 = "challenges"
-// Expected output: 2
-
-
-// b) Create the function that makes the test pass.
+// input: a word
+// output: the first vowels index
+// input: a number
+// output: <number> is not a number
+
+describe("firstVowel", () => {
+  it("function passes in a string and returns the first vowels index", () => {
+    
+    const vowelTester1 = "learn"
+    const vowelTester2 = "academy"
+    const vowelTester3 = "challenges"
+    
+    expect(firstVowel(vowelTester1)).toEqual(1)
+    // Expected output: 1
+    
+    expect(firstVowel(vowelTester2)).toEqual(0)
+    // Expected output: 0
+    
+    expect(firstVowel(vowelTester3)).toEqual(2)
+    // Expected output: 2
+    
+    expect(firstVowel(5)).toEqual('5 is not a word')
+    
+  })
+})
+
+// ReferenceError: firstVowel is not defined
+
+//b) Create the function that makes the test pass.
+
+// Pseudo----->
+// create a function called firstVowel
+// pass in a <string> paramater to the function
+// use a conditional if statement to evaluate if the typeof the parameter <string> is strictly equal to 'string'
+// if the statement evaluates to true return the paramater and access the string using the .search() method
+// in the search method use a regular expression that contains all vowels followed by the character i to search for both upper and lowercase letters.
+// if the statement returns false use an else statement to return `${str} is not a word`
+// if outside of jest testing log function to see result
+
+ const firstVowel = (str) =>{ 
+    if(typeof str === 'string'){
+      return str.search(/[aeiou]/i)
+    }else{
+      return `${str} is not a word`
+    }
+ }
```

Comment:
Outstanding work!  Great research on using the search method and providing detailed pseudocode that explains wonderfully on what is happening there.  I want to encourage you to share more of these answers when we go through the assessment together in class.